content/tutorials/MQTT-snake-js.textile

```diff
@@ -0,0 +1,267 @@
+---
+title: "MQTT Tutorial: Snake with Javascript"
+section: tutorials
+index: 104
+languages:
+  - javascript
+  - nodejs
+---
+
+Ably Realtime provides support for a number of protocols with its pub/sub system, one of which is MQTT. MQTT is an open ISO standard providing a lightweight messaging protocol for small sensors and mobile devices, optimized for high-latency or unreliable networks. Whilst we recommend use of the Ably realtime protocol where you can, MQTT can be great for use with languages we don't currently support, as well as when you stringent bandwidth restrictions.
+
+h2. What you'll be building
+
+In this tutorial, you'll be using the MQTT protocol with Ably to connect a controller made in Node.js with a game of snake to be played in a browser. This tutorial will be doing this with the assumption that your input will be from a keyboard. The language or device used as the controller isn't important however, so long as MQTT is supported.
```

Comment:
If we're going for an mqtt tutorial using a language we have a native client for (such as node), we should make very clear that it's just for demo purposes, and if someone actually wanted to do this, they should use ably-js, not mqtt. (Ideally we'd do it it a language that we don't have a native client for, so that the tutorial would be an actual use-case, to avoid that confusion, but if we're not doing that, but shrug)

Reply:
Good point, I'll add the clarification for JavaScript now. I'm also going to try and make a few language options for this tutorial, but we have this JavaScript one as an easy 'get going easily' version. Do you have any preferences on languages we should also have this tutorial on?